syntax = "proto3";

package basic_package;

message Data1 {
  bool value1 = 1;
  string value2 = 2;
  int32 value3 = 3;
  int64 value4 = 4;
  float value5 = 5;
}
